nekoyume/Assets/_Scripts/UI/Widget/Popup/StakingPopup.cs

```diff
@@ -0,0 +1,107 @@
+using System.Linq;
+using Nekoyume.Game.Controller;
+using Nekoyume.Helper;
+using Nekoyume.Model.Item;
+using Nekoyume.State;
+using Nekoyume.UI.Module;
+using TMPro;
+using UniRx;
+using UniRx.Triggers;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Nekoyume.UI
+{
+    public class StakingPopup : PopupWidget
+    {
+        [SerializeField] private Button closeButton;
+
+        [SerializeField] private ConditionalButton stakingButton;
+
+        [SerializeField] private Image stepImage;
+
+        [SerializeField] private TextMeshProUGUI stepText;
+
+        [SerializeField] private TextMeshProUGUI stepScoreText;
+
+        [SerializeField] private StakingItemView[] regularItemViews;
+
+        [SerializeField] private GameObject tooltip;
+
+        [SerializeField] private RectTransform tooltipRectTransform;
```

Comment:
Are `GameObject tooltip` and `RectTransform tooltipRectTransform` component of diffrent objects?
If it is same... can remove `GameObject tooltip`.

Reply:
I applied this, Thank you!